### DISCUSSION TITLE
Quarto not identifying wrong YAML options

### DISCUSSION THREAD

**Discussion by dense-set** [quarto-dev/quarto-cli#5873] [Q&A] [answered]
### Description

(Not sure if this is a (even a?) bug of Quarto or something else, hence asking in Q&A.)

I am learning to use Quarto. Playing around, I discovered that passing wrong arguments to the header YAML sometimes gets undetected and the file still renders. For instance, with `highlight-style: bad123`, the following still renders:

````qmd
---
title: test
highlight-style: bad123
jupyter: python3
---

```{python}
import numpy as np

a = np.arange(15).reshape(3, 5)
a.shape
```
````

However, if I mess up the argument of `jupyter`, I get greeted with an error.

Why is so? Is this expected behaviour?

**Comment by mcanouil** [comment]
`highlight-style` accepts custom style which can have anything as name, see <https://quarto.org/docs/output-formats/html-code.html#custom-highlighting>.

**Reply by dense-set** [reply]
Even if a `.theme` file, or any file of the provided name, is not found (in the project directory)?

**Reply by mcanouil** [reply, marked as answer]
I opened an issue to improve this as Pandoc does error out if when a file, it's not ending with `.theme`.
See #5881.
Thanks for the report.
Closing this thread.
